transitmatters/t-performance-dash
# fix: alert annotations not applied for travel times graph on station change

## Motivation
Closes https://github.com/transitmatters/t-performance-dash/issues/801 



## Changes

https://github.com/transitmatters/t-performance-dash/assets/46619169/8231a6b5-dbd8-40bd-8d63-a2be11212bcd





## Testing Instructions

1. Add an alert on the /[line]/trips/single page
2. Switch the "to" station 
3. Confirm that the alert persists

Note that this change will persist the alert even if the user changes routes within the line. The alert will be persisted until the user clicks remove.
